lua/chatgpt/api.lua

```diff
@@ -4,10 +4,12 @@ local logger = require("chatgpt.common.logger")
 
 local Api = {}
 
+-- API Host please no http or https
+Api.OPENAI_API_HOST = os.getenv("OPENAI_API_KEY") or "api.openai.com"
```

Comment:
is this should be `os.getenv("OPENAI_API_HOST")` ?

Reply:
Yes, thanks.